### PR TITLE
[Blog] Cost of Observability

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit adafbc3b2c6c4922a5f827c9270a2e5abe0203e5 at: https://663efc3ae261e59eb976f9d5--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@leecalcote is this ready? We cam merge this?

**Comment by leecalcote** [comment]
Thanks for asking. It's _almost_ there.

**Comment by l5io** [comment]
🚀 Preview for commit b4bbabb1915045952486fd6c8130ec677d71307c at: https://665811ae1ce667a4c02f8b0f--layer5.netlify.app

File: src/collections/blog/2024/05-12-cost-of-observability/index.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import { Link } from "gatsby";
+import comparativeCost from "./comparative-cost-of-observability-signals.png";
+import Blockquote from "../../../../reusecore/Blockquote";
+
+<BlogWrapper>
+
+The world of software development is embroiled in a silent war – the Observability Arms Race.  On one side, verbose logs, relics of our debugging I'm hoping that we can give our mascot one heck of a rocket ride. https://www.linkedin.com/feed/update/urn:li:activity:7089583821129482241days, threaten to devour production budgets.  On the other, traces, the Sherlock Holmes of observability, offer targeted insights but come with their own complexities.
```
Comment (leecalcote) [inline src/collections/blog/2024/05-12-cost-of-observability/index.mdx:25]:
```suggestion
The world of software development is embroiled in a silent war – the Observability Arms Race.  On one side, verbose logs, relics of our debugging days, threaten to devour production budgets.  On the other, traces, the Sherlock Holmes of observability, offer targeted insights but come with their own complexities.
```

**Comment by singh1203** [comment]
Impressive opening! The subtle Sherlock Holmes theme is a fantastic touch.

**Comment by l5io** [comment]
🚀 Preview for commit bc8b7470b3ceba402c9c59d0d1592387615a1fb8 at: https://66621ec6e0c9f6631730b31a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 32d2b654609fc16d14f43cf06f77b0d9468f4eb3 at: https://667f361ffc8ca0a776714d69--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 51f5bab69b6d94775aceb1efba1687aed3af6513 at: https://66a00f69662a122a26865fcf--layer5.netlify.app

**Pull request merged**